src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.hasOwnProperty('age')
+      ? (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      ) : null}
+
+    {person.isMarried === false
+      ? (
+        <p className="Person__partner">
+          I am not married
+        </p>
+      ) : null}
+
+    { person.hasOwnProperty('partnerName') && person.sex === 'f' && (
+    <p className="Person__partner">
+      {`${person.partnerName} is my husband`}
+    </p>
+    )}
+
+    { person.hasOwnProperty('partnerName') && person.sex === 'm' && (
+    <p className="Person__partner">
+      {`${person.partnerName} is my wife`}
+    </p>
+    )}
```

Comment:
Marriage text move to variable and combine tags into 1. It should be looks like this 
```suggestion
    <p className="Person__partner">
        {person.isMarried
          ? `${person.partnerName} is my ${yourAwesomeVariableName}`
          : 'I am not married'}
      </p>
    )}
```